cmd/brimcap/analyze/command.go

```diff
@@ -58,19 +62,25 @@ func (c *Command) Run(args []string) (err error) {
 	}
 	defer emitter.Close()
 
-	c.analyzer, err = c.analyzeflags.Open(ctx, args)
+	pcapfile, pcapsize, err := cli.OpenFileArg(args[0])
 	if err != nil {
 		return err
 	}
+	defer pcapfile.Close()
+
+	display := analyzecli.NewDisplay(c.JSON, pcapsize)
+	zctx := resolver.NewContext()
+	analyzer := analyzer.CombinerWithContext(ctx, zctx, pcapfile, c.analyzeflags.Configs...)
 
 	// If not emitting to stdio write stats to stderr.
 	if c.out.FileName() != "" {
-		c.analyzer.RunDisplay()
+		display.Run(analyzer)
```

Comment:
Maybe do everything with `display` here?
```suggestion
		display := analyzecli.NewDisplay(c.JSON, pcapsize)
		display.Run(analyzer)
		defer display.Close()
```

Reply:
That'll work!